README.md

```diff
@@ -26,23 +26,51 @@ Most of Data.gov discussions happen at [Data.gov github](https://github.com/gsa/
 - [Docker](https://docs.docker.com/install/) v18+
 - [Docker Compose](https://docs.docker.com/compose/) v1.24+
 
+
 ### Setup
 
 Build the docker containers.
 
-    $ docker-compose build
-
-Run the docker containers.
-
-    $ docker-compose up
+    $ make build
 
-Activate all the installed plugins and theme.
+Configure WordPress by installing the composer dependencies, installing
+WordPress, and configuring plugins and theme.
 
-    $ docker-compose exec app wp core install --url=http://localhost:8000 \
-      --title=Data.gov --admin_user=admin --admin_email=admin@example.com --allow-root
-    $ docker-compose exec app wp plugin activate --all --allow-root
-    $ docker-compose exec app wp theme activate roots-nextdatagov --allow-root
+    $ make configure
 
 Open your browser to [localhost:8000](http://localhost:8000/).
 
 _TODO: initialize the database with seed data so the theme loads properly._
+
+Once the containers are setup, you can just bring up the containers without
+running `make setup` each time.
+
+    $ make up
+
+If you need to start from a clean state, remove any containers and volumes.
+
+    $ make clean
+
+
+### Restoring database dumps
+
+You don't need a database dump for most development tasks. If you need
+a database dump, you can create one following instructions from the
+[Runbook](https://github.com/GSA/datagov-deploy/wiki/Runbook#database-dump).
+
+Once you have the database dump, you can restore it for your local development
+environment.
+
+    $ docker-compose exec -T db mysql -u root -pmysql-dev-password datagov \
+      < <(gzip --to-stdout --decompress databasedump.sql.gz)
+
+
+### Tests
+
+Make sure the app is setup and WordPress is installed.
+
+    $ make setup
```

Comment:
There's no `setup` target... Did you mean `up`?

Reply:
This should be `make configure`.